index.js

```diff
@@ -0,0 +1,87 @@
+const express = require('express');
+const cors = require('cors');
+const knex = require('knex');
+const knexConfig = require('./knexfile.js')
+const bcrypt = require('bcryptjs');
+const session = require('express-session');
+const secret = "secret";
+const KnexSessionStore = require('connect-session-knex')(session);
+const db = knex(knexConfig.development);
+const server = express();
+const jwt = require('jsonwebtoken')
+server.use(express.json());
+server.use(cors());
+
+function generateToken(user) {
+    
+    const payload = {
+        username: user.username
+    }
+
+    const options = {
+        expiresIn: '1h',
+        jwtid: '56789',
+    };
+
+    return jwt.sign(payload, secret, options);
+}
+
+function protected(req, res, next) {
+const token = req.headers.authorization;
```

Comment:
Indentation inside your function blocks would help a lot in terms of readability.